testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/shared/SharedSessionFailoverTestCase.java

```diff
@@ -57,6 +57,7 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
+@org.junit.Ignore("WFLY-9428")
```

Comment:
the issue does not look related to messaging-activemq changes, right?

Reply:
It is a consequence of https://issues.jboss.org/browse/WFLY-9229.
The root cause of WFLY-9428 is more complex - and is addressed by https://github.com/wildfly/wildfly/pull/10665/commits/b0da6886d0f4ac3e464d3a9624908829db3fe1f6